### PR TITLE
Edge file constructor fix

### PR DESCRIPTION
**Pull request by jakearchibald**
**Do not merge**. Not based on master.

### CODE REVIEW

**Comment by jakearchibald** [comment]
@surma still happy?

File: src/components/App/index.tsx [hunk 1]
```diff
@@ -275,7 +276,7 @@ export default class App extends Component<Props, State> {
 
     const image = images[index];
 
-    let file: File | undefined;
+    let file: File | Fileish | undefined;
```
[the remark this answers is not in the document]
Comment (jakearchibald) [reply src/components/App/index.tsx]: Nah, it might be the source file, which is always a genuine `File`.

[the remark this answers is not in the document]
Comment (surma) [reply src/components/App/index.tsx]: Maybe Fileish should be an interface and the class is just used when necessary?

But that’s probably a lot of work to save one type union. Go ahead and merge :D

[the remark this answers is not in the document]
Comment (jakearchibald) [reply src/components/App/index.tsx]: If we need to do the union in a few other places, happy to change it.

**Pull request merged**